docs/source/installation/linux.rst

```diff
@@ -17,6 +17,17 @@ Install manim via pypi::
     # pip3 install manimlib
 
 OR Install manim via the git repository with venv::
+NOTE : Make sure that the python version you are using is python3.7 because manim works only for that version . If not, you will get errors when running the old_projects.If the python3 version is not 3.7 then upgrade your version usimg the following steps:
+     
+sudo apt-get install -y software-properties-common
+sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
This should probably only be given as a suggestion. Paranoid users may prefer downloading the source from python.org.

Reply:
Yeah ! So, can I add that statement that this is one of the various methods available or should I also give instructions for downloading from python.org?

Reply:
Just say to use python3.7 and offer both options as suggestions.